guide/C/ch_bus_features.xml

```diff
@@ -555,30 +548,29 @@
         </screenshot>
       </figure>
 
-      <para>Here you can:
-      </para>
+      <para>Perform the following procedure in order to complete it.</para>
 
-      <itemizedlist>
-        <listitem>
-          <para>Enter the name of your company along with contact information such as your phone number, fax number,
-            e-mail address and website URL.
-          </para>
-        </listitem>
+      <procedure>
+        <step><simpara>Enter appropriate information into <guilabel>Company Name</guilabel>, <guilabel>Company Address</guilabel>,
+            <guilabel>Company Contact Person</guilabel>, <guilabel>Company Phone Number</guilabel>, <guilabel>Company Fax Number</guilabel>,
+            <guilabel>Company Email Address</guilabel>, and <guilabel>Company Website URL</guilabel>.
+          </simpara>
+        </step>
 
-        <listitem>
-          <para>Enter your company&rsquo;s tax payer id number in the <guilabel>Company ID</guilabel> field.
-          </para>
-        </listitem>
+        <step><simpara>Enter your company&rsquo;s tax payer id number in the <guilabel>Company ID</guilabel> field.</simpara></step>
+      
+        <step><simpara>Select <guilabel>Default Customer Tax Table</guilabel> and <guilabel>Default Vendor Tax Table</guilabel>
```

Comment:
In the GnuCash msgid, the word "TaxTable" is used. But it should be replaced with "Tax Table" for unification.